fdbserver/workloads/MetaclusterManagementWorkload.actor.cpp

```diff
@@ -401,11 +401,12 @@ struct MetaclusterManagementWorkload : TestWorkload {
 			state std::vector<TenantState> filters;
 			filters.push_back(checkState);
 			state std::vector<std::pair<TenantName, TenantMapEntry>> tenantList;
-			// Possible to have changed state between now and the getTenant call above
 			state TenantMapEntry checkEntry2;
-			wait(store(checkEntry2, MetaclusterAPI::getTenant(self->managementDb, tenant)) &&
-			     store(tenantList,
+
+			wait(store(tenantList,
 			           MetaclusterAPI::listTenantMetadata(self->managementDb, ""_sr, "\xff\xff"_sr, 10e6, 0, filters)));
+			// Possible to have changed state between now and the getTenant call above
+			wait(store(checkEntry2, MetaclusterAPI::getTenant(self->managementDb, tenant)));
```

Comment:
Since we aren't running them concurrently, we can more succinctly write these with a direct assignment:

```suggestion
			TenantMapEntry checkEntry2 = wait(MetaclusterAPI::getTenant(self->managementDb, tenant));
```